src/textual/widgets/_data_table.py

```diff
@@ -55,10 +53,20 @@ class Column:
     """Table column."""
 
     label: Text
-    width: int
+    width: int = 0
     visible: bool = False
     index: int = 0
 
+    content_width: int = 0
+    auto_width: bool = False
+
+    @property
+    def render_width(self) -> int:
+        if self.auto_width:
```

Comment:
Could you add a comment to the code saying what the `+ 2` is for future readers?